src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -27,6 +27,14 @@
 
 "Common_Alert_Action_moreInfo" = "Mehr Erfahren";
 
+"Common_BackgroundFetch_AlertMessage" = "Sie haben die Hintergrundaktualisierung für die Corona-Warn-App ausgeschaltet. Um die automatische Risiko-Ermittlung zu nutzen, müssen Sie die Hintergrundaktualisierung einschalten. Wenn Sie die Hintergrundaktualisierung nicht einschalten, können Sie Ihre Risiko-Ermittlung nur manuell in der App starten.
+
+Sie können die Hintergrundaktualisierung für die App jederzeit in den Geräteeinstellungen einschalten.";
+
+"Common_BackgroundFetch_SettingsTitle" = "Geräteeinstellungen öffnen";
+
+"Common_BackgroundFetch_OKTitle" = "Risiko-Ermittlung manuell starten";
```

Comment:
For English locale strings will be provided later by translators?

Reply:
I will double check with @steins-code on this one. Thanks for addressing this.

Reply:
I have added in placeholders for the keys. According to @steins-code, the translators will double-check the texts and translate them to the corresponding languages.